rundeckapp/grails-app/views/execution/mailNotification/status.gsp

```diff
@@ -1,378 +1,785 @@
+                              <g:set var="summary" value="${vals[1]+' of '+vals[2]}"/>
+                              </g:if>
+                          </g:if>
+                          <g:else>
+                              <g:set var="perc" value="${0}"/>
+                          </g:else>
+                          <g:if test="${vals && vals.size()>1 && vals[1]!='0' && vals[1]!=0}">
+                              <g:enc>${vals[1]}</g:enc> failed
+                          </g:if>
+                          <g:else>
+                              <g:enc>${vals[0]}</g:enc> ok
+                          </g:else>
+                          <g:if test="${perc>0}">
+                          <g:render template="/common/progressBar" model="${[completePercent:(int)perc,title:'Failed nodes',className:'nodes failure',showpercent:false,innerContent:summary]}"/>
+                          </g:if>
+                  </g:if>
+                  <g:if test="${execution?.failedNodeList}">
+                          <g:set var="failednodes" value="${execution?.failedNodeList.split(',')}"/>
+                          <g:if test="${!nodestatus}">
+                          <g:enc>${failednodes.length}</g:enc> failed:
+                          </g:if>
+                          <div><g:enc>${execution?.failedNodeList}</g:enc></div>
+                  </g:if>
+                </td>
+              </tr>
             </table>
-
-                    </td>
-                    <g:if test="${scheduledExecution && renderJobStats}">
-                        <td style="vertical-align:top;" class="toolbar small">
-                            <g:render template="/scheduledExecution/renderJobStats"
-                                      model="${[scheduledExecution: scheduledExecution]}"/>
-                        </td>
-                    </g:if>
+          </div>
+        </div>
+      </td>
+    </tr>
+    <tr>
+      <td class="spacer py-sm-20" height="20"></td>
+    </tr>
+    </g:if>
+    <g:if test="${logOutput}">
+    <tr>
+      <td class="col" colspan="2" style="padding: 0 10px;">
+        <div>
+          <a class="toggle-trigger" style="text-decoration: none;">
+            <button class="toggle-trigger"
+              style="background-color: #EEEEEE; margin: 0; padding: 0; display: block; width: 100%; text-align: left; border: none; outline: none; font-size: 15px;">
+              <table bgcolor="#EEEEEE" cellpadding="0" cellspacing="0" role="presentation" width="100%">
+                <tr>
+                  <td style="padding: 20px;">Log Output</td>
                 </tr>
+              </table>
+            </button>
+          </a>
+          <div class="toggle-content">
+            <table cellpadding="0" cellspacing="0" width="100%" role="presentation" style="border: 1px solid #EEEEEE;">
+              <tr>
+                <td style="padding: 20px;">
+                  ${enc(sanitize:logOutput)}
```

Comment:
output should go inside a <pre> tag so that line endings from the output show up correctly

Reply:
I've added a `<pre>` tag around the log output.